cookbooks/aws-parallelcluster-environment/kitchen.environment-config.yml

```diff
@@ -734,3 +734,15 @@ suites:
       dependencies:
         - recipe:aws-parallelcluster-platform::directories
         - resource:spack:setup
+  - name: default_user_local_home
+    run_list:
+      - recipe[aws-parallelcluster-tests::setup]
+      - recipe[aws-parallelcluster-environment::config_default_user_home]
+    verifier:
+      controls:
+        - local_default_user_home
+    attributes:
+      cluster:
+        node_type: 'HeadNode'
```

Comment:
Since the setting should be applied to every cluster node, shouldn't we test also on compute and login nodes?

Reply:
We actually can't test on compute and login nodes because it depends on a shared directory, but we can add integ tests for these cases